docs/dev_guide/contents/newcomers.rst

```diff
@@ -162,7 +162,7 @@ This will make submitting changes easier in the long term for you:
 
     $ git clone https://github.com/<username>/sunpy.git sunpy-git
     $ cd sunpy-git
-    $ pip install -e .[dev]
+    $ pip install -e ."[dev]"
```

Comment:
```suggestion
    $ pip install -e ".[dev]"
```

Can you check this works?

Reply:
That does work and is more consistent with the other docs.